taskana-adapter-taskana-connector/src/main/java/pro/taskana/adapter/taskanaconnector/api/impl/TaskanaSystemConnectorImpl.java

```diff
@@ -180,15 +173,15 @@ public void terminateTaskanaTask(ReferencedTask referencedTask)
 
     for (TaskSummary taskSummary : requestedTasks) {
       try {
-        Task taskanaTask = taskService.getTask(taskSummary.getTaskId());
+        Task taskanaTask = taskService.getTask(taskSummary.getId());
         Map<String, String> callbackInfo = taskanaTask.getCallbackInfo();
         if (callbackInfo != null
             && callbackInfo.get(REFERENCED_TASK_ID) != null
             && callbackInfo.get(SYSTEM_URL) != null) {
           result.add(taskInformationMapper.convertToReferencedTask(taskanaTask));
         }
       } catch (TaskNotFoundException | NotAuthorizedException e) {
-        LOGGER.error("Caught {} when trying to retrieve requested taskana tasks.", e);
+        LOGGER.error("Caught {} when trying to retrieve requested taskana tasks.", e, e);
```

Comment:
Neither this nor the original version is the intended usage of SLF4J with Exceptions. See my post in the slack taskana channel. But since this problem occurs through Taskana and adapter, we should treat it with a separate ticket....